app/views/posts/_expanded.html.erb

```diff
@@ -447,8 +448,10 @@
                 <% elsif post.comments_disabled %>
                   <p class="has-color-red-500"><i class="fa fa-lock"></i> Comments have been disabled on this post, but as a moderator you are exempt from that block.</p>
                 <% end %>
-                <a href="#" class="button is-muted is-small" data-modal=".js--<%= post.id %>-new-thread-modal">Start new comment thread</a>
+<!--                <a href="#" class="button is-muted is-small" data-modal=".js--<%#= post.id %>-new-thread-modal">Start new comment thread</a>-->
```

Comment:
If you don't need the previous definition any more, you can remove this comment.